### PR TITLE
chore: bump dependencies

### PR DESCRIPTION
**Pull request by Unix4ever**
Via renovate bot.

Signed-off-by: Artem Chernyshev <artem.chernyshev@talos-systems.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -100,7 +100,7 @@ require (
 	github.com/siderolabs/siderolink v0.2.0
 	github.com/spf13/cobra v1.6.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.8.0
```
Comment (smira) [inline go.mod:103]: there _should_ be more here, as at least AWS SDK should be bumped.

`make go-mod-outdated` lists everything

Comment (Unix4ever) [reply go.mod:103]: should be good now

Comment (smira) [reply go.mod:103]: I don't see any changes

**Review by smira — APPROVED** [review]

**Pull request merged**